spec/requests/api/v1/bracks/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe "Bracks API" do
+  attr_reader :first_closest, :first_furthest, :second_closest, :second_furthest
```

Comment:
Yeah - I was just trying to validate that the racks I think were closets and further for each test are valid - but your suggestion is a better way to do it! This is just weird!